specs/AgriFood/AgriCrop/example.jsonld

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
we would need examples as well for NGSIv2, both normalized and keyValues

Reply:
Do I also need to implement a validator for NGSIv2?